indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -133,3 +134,13 @@ def _process(self):
             logger.info("Registrations for {} scheduled by {}".format(self.regform, session.user))
             return jsonify_data(flash=False)
         return jsonify_template('events/registration/management/regform_schedule.html', form=form)
+
+
+class RHModifyRegForm(RHManageRegFormBase):
```

Comment:
I'd keep consistency in the naming here: `RHRegistrationFormFormModify`. I know it's ugly, but it comes with the fact that we are calling the very component `RegistrationForm` and that within it there is an actual `Form`. I suggested a name change to @pferreir in the same fashion of `Survey` containing a `Questionnaire`, but he's still not convinced.